app/javascript/components/ansible-credentials-form/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useMemo, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import MiqFormRenderer from '@@ddf';
+import createSchema from './ansible-credential-form.schema';
+import { initNotifications } from '../../miq-redux/actions/notifications-actions';
```

Comment:
Remove this please, probably your editor's autocomplete is a little bit crazy :see_no_evil: